Formula/gssdp.rb

```diff
@@ -51,10 +47,10 @@ def install
       -I#{gettext.opt_include}
       -I#{glib.opt_include}/glib-2.0
       -I#{glib.opt_lib}/glib-2.0/include
-      -I#{include}/gssdp-1.2
+      -I#{include}/gssdp-1.6
```

Comment:
Shouldn't this be `#{version.major_minor}`?

Reply:
Yea, maybe. Never really sure with GNOME. This is meant to be the API version, and sometimes it seems to be the same as the formula version (e.g. here), and sometimes it's not (e.g. glib).